docs/meshstack.azure.landing-zones.md

```diff
@@ -109,13 +113,38 @@ Blueprint assignments are performed with a specific [resource locking mode](http
 | AllResourcesReadOnly    | Locked resource groups are read only and other locked resources can't be modified at all. |
 | AllResourcesDoNotDelete | Locked resources can be modifiede but not deleted.                                        |
 
-## meshRole to Platform Role Mapping
+### meshRole to Platform Role Mapping
+
+Both the Subscription and Resource Group based integration supports this Landing Zone configuration option.
+
+Please see [meshRole to Platform Role Mapping](#platform-role-mapping).
+
+## Azure Resource Group
+
+The following description targets the Azure integration based on Resource Groups.
+
+### Resource Group Location
+
+The newly created Resource Group for the meshProjects will get assigned to this location. Please take care that the entered location must be all lower case and without spaces (e.g. eastus2 for East US 2).
+In order to list the available locations you can use `az account list-locations --query "[*].name" --out tsv | sort`.
+
+### meshRole to Platform Role Mapping
+
+Both the Subscription and Resource Group based integration supports this Landing Zone configuration option.
+
+Please see [meshRole to Platform Role Mapping](#platform-role-mapping).
+
+## Shared Configuration
+
+This options are shared between the two different Landing Zone configuration.
```

Comment:
```suggestion
These options are shared between both Azure Subscription & Azure Resource Group configuration.
```

Reply:
needed to shift some sections around in order to make the linter happy.
added ur suggestion